services/etcd.go

```diff
@@ -281,6 +282,7 @@ func RunEtcdSnapshotSave(ctx context.Context, etcdHost *hosts.Host, prsMap map[s
 	if err := docker.DoRunContainer(ctx, etcdHost.DClient, imageCfg, hostCfg, EtcdSnapshotContainerName, etcdHost.Address, ETCDRole, prsMap); err != nil {
 		return err
 	}
+	time.Sleep(EtcdSnapshotWaitTime * time.Second)
```

Comment:
I feel like we might need to add validation for snapshot related parameters prior to starting a container. But at this point, waiting might be the least impactful way in terms of qa. Let me know what you think